src/instruction.rs

```diff
@@ -3,12 +3,14 @@ pub enum Instruction {
     Add,
     Mul,
     Push { value: u64 },
+    PrintLn,
 }
 
 impl PartialEq for Instruction {
     fn eq(&self, other: &Instruction) -> bool {
         match (self, other) {
             (Instruction::Add, Instruction::Add) => true,
+            (Instruction::PrintLn, Instruction::PrintLn) => true,
```

Comment:
I wonder if we can derive `PartialEq` for `Instruction`?

Reply:
yeah you're right, I can replace it with `#[derive(Debug, PartialEq)]`

Reply:
changed :point_right:  4873a468efcb289f99739a9346701219f808fa83